Dockerfile

```diff
@@ -5,22 +5,42 @@ ADD super_netperf   /sbin/
 ADD percpu_netperf  /sbin/
 ADD latency_netperf /sbin/
 
-RUN											   \
-	apk add --update curl wrk iperf3 git build-base texinfo bash iproute2 automake     \
-			 libtool intltool autoconf					&& \
-	git clone https://github.com/HewlettPackard/netperf.git				&& \
-	cd netperf/									&& \
-	./autogen.sh									&& \
-	chmod a+x configure								&& \
-	./configure --prefix=/usr							&& \
-	make										&& \
-	make install									&& \
-	cd ..										&& \
-	rm -rf netperf									&& \
-	rm -f /usr/share/info/netperf.info						&& \
-	strip -s /usr/bin/netperf /usr/bin/netserver					&& \
-	apk del build-base texinfo git automake libtool intltool autoconf		&& \
-	rm -rf /var/cache/apk/*
+# The CFLAGS=-fcommon - https://gcc.gnu.org/gcc-10/porting_to.html
+# Fixes: multiple definition of `loc_nodelay'; nettest_bsd.o:./src/nettest_bsd.c:206: first defined here
+RUN	apk add --update \
+	curl \
+	wrk \
+	git \
+	build-base \
+	texinfo \
+	bash \
+	iproute2 \
+	automake \
+	libtool \
+	intltool \
+	autoconf \
+	lksctp-tools-dev \
+	linux-headers \
+	&& git clone https://github.com/HewlettPackard/netperf.git \
```

Comment:
Maybe we should pin our image to a specific netperf commit while we are it? (That would warrant its own commit.)

Reply:
> Thanks, looks good to me!
> 
> Could you merge the two commits please?

Sure! One question regarding this: what do you usually prefer during the PR? Push the patches based on the reviews  and in the end merge everything or do you prefer overwrite the last commit in every time I submit a change?

Reply:
Overwrite the last commit. It's best to keep the commit history always clean so that, if it looks good, we can merge directly. If we want to see what changed, we can always click on the `force-pushed` links in the GitHub UI.